doc/source/ray-core/ray-dashboard.rst

```diff
@@ -82,57 +314,96 @@ The information of up to 1000 dead actors will be stored.
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/actor-detail.png
     :align: center
 
-Metrics View
-~~~~~~~~~~~~~
-The metrics view lets you view visualizations of the time series metrics emitted by Ray. It requires that prometheus and grafana is running for your cluster.
-Instructions about that can be found :ref:`here <ray-metrics>`.
+From the actor detail page, you can see the metadata, state, and the all tasks that have run from this actor. 
 
-You can select the time range of the metrics in the top right corner. The graphs refresh automatically every 15 seconds.
+.. _dash-metrics-view:
 
-There is also a convenient button to open the grafana UI from the dashboard. The Grafana UI provides additional customizability of the charts.
+Metrics View
+------------
 
 .. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/metrics.png
     :align: center
 
-Event View
-~~~~~~~~~~
+Ray exports default metrics which are available from the :ref:`Metrics View <dash-metrics-view>`. Here are some available example metrics.
+
+- The tasks, actors, and placement groups broken down by states.
+- The :ref:`logical resource usage <logical-resources>` across nodes.
+- The hardware resource usage across nodes.
+- The autoscaler status.
+
+See :ref:`System Metrics Page <system-metrics>` for available metrics.
 
 .. note:: 
 
-  The event view feature is experimental.
+  The metrics view required the Prometheus and Grafana setup. **See :ref:`Ray Metrics <ray-metrics>` to learn how to set up Prometheus and Grafana**.
 
-The event view lets you see a list of events associated with a specific type (e.g., autoscaler or job) in a chronological order. The equivalent information is also accessible via CLI commands ``ray list cluster-events`` :ref:`(Ray state APIs)<state-api-overview-ref>`.
+The metrics view lets you view visualizations of the time series metrics emitted by Ray.
 
-There are 2 types of events that are available.
+You can select the time range of the metrics in the top right corner. The graphs refresh automatically every 15 seconds.
 
-- **Job**: Events related to :ref:`Ray job submission APIs <jobs-quickstart>`.
-- **Autoscaler**: Events related to :ref:`Ray autoscaler <cluster-autoscaler>`
+There is also a convenient button to open the grafana UI from the dashboard. The Grafana UI provides additional customizability of the charts.
+
+.. _dash-logs-view:
 
-.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/event.png
+Logs view
+---------
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/logs.png
     :align: center
 
+The logs view lets you view all the ray logs that are in your cluster. It is organized by node and log file name. Many log links in the other pages will link to this view and filter the list so the relevant logs appear.
```

Comment:
Can we add a link to the logging structure in the ray logging documentation? One platform user didn't check out the logging structure page and had no idea how to persist the right log files for the right end users. Adding a link here may help users find it.